sources/packages/web/src/components/common/notes/StudentNotes.vue

```diff
@@ -5,6 +5,7 @@
         <template #actions>
           <v-btn-toggle
             v-model="toggleNotes"
+            density="compact"
```

Comment:
is this communicated to @CarlyCotton and business?. As we may have a number of these buttons all around which can be set to density compact and that can be done with a small ticket.

Reply:
I mentioned in our conversation during the meeting where I asked for feedback about the UI. However it is not written anywhere so i'll ask Carly to include this tomorrow.